lib/metrics/gc.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const Counter = require('../counter');
+const Histogram = require('../histogram');
+
+let perf_hooks;
+
+try {
+	// eslint-disable-next-line
+	perf_hooks = require('perf_hooks');
+} catch (e) {
+	// node version is too old
+}
+
+const NODEJS_GC_RUNS_TOTAL = 'nodejs_gc_runs_total';
+const NODEJS_GC_DURATION = 'nodejs_gc_duration';
```

Comment:
I think the naming convention here would be `nodejs_gc_duration_seconds`,  like `http_request_duration_seconds` from https://prometheus.io/docs/practices/naming/#metric-names

Reply:
fixed